### PR TITLE
[No Ticket] Bump dependencies

### PR DESCRIPTION
**Pull request by broadbot**
Consolidate scala-steward PRs

### CODE REVIEW

**Comment by rtitle** [comment]
```
11:13:57 [warn] 	Note: Unresolved dependencies path:
11:13:57 [error] sbt.librarymanagement.ResolveException: Error downloading org.scalameta:semanticdb-scalac_2.13.6:4.4.10
11:13:57 [error]   Not found
11:13:57 [error]   Not found
11:13:57 [error]   not found: /root/.ivy2/localorg.scalameta/semanticdb-scalac_2.13.6/4.4.10/ivys/ivy.xml
11:13:57 [error]   not found: https://repo1.maven.org/maven2/org/scalameta/semanticdb-scalac_2.13.6/4.4.10/semanticdb-scalac_2.13.6-4.4.10.pom
11:13:57 [error]   not found: https://broadinstitute.jfrog.io/broadinstitute/libs-release/org/scalameta/semanticdb-scalac_2.13.6/4.4.10/semanticdb-scalac_2.13.6-4.4.10.pom
11:13:57 [error]   not found: https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot/org/scalameta/semanticdb-scalac_2.13.6/4.4.10/semanticdb-scalac_2.13.6-4.4.10.pom
11:13:57 [error] 	a
```

**Comment by rtitle** [comment]
Rebased.. hoping `scalameta` adds support for 2.13.6 soon

File: project/Settings.scala [hunk 1]
```diff
@@ -26,7 +26,10 @@ object Settings {
     Test / console / scalacOptions --= Seq("-Ywarn-unused:imports", "-Xfatal-warnings"), //disable unused fatal warning in `sbt test:console`
     Test / scalacOptions --= List("-Ywarn-dead-code", "-deprecation", "-Xfatal-warnings"),
     addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
-    addCompilerPlugin(scalafixSemanticdb)
+    // Hard-coding the semanticdbVersion to make it work with Scala 2.13.6. It's possible this may be fixed
+    // in a future scalafix version. See: https://scalacenter.github.io/scalafix/docs/users/installation.html
+    semanticdbEnabled := true,
```
Comment (rtitle) [inline project/Settings.scala:31]: This looks like the right way to install `scalafix` for sbt 1.3* and newer based on https://scalacenter.github.io/scalafix/docs/users/installation.html. 

Although I had to hard-code the `semanticdb` version to work with Scala 2.13.6: https://mvnrepository.com/artifact/org.scalameta/semanticdb-scalac

**Review by Qi77Qi — APPROVED** [review]

**Comment by rtitle** [comment]
Unit test error:
```
(update) sbt.librarymanagement.ResolveException: Error downloading org.scoverage:scalac-scoverage-plugin_2.13.6:1.4.6
```
Looks like there is no `2.13.6` version of `scalac-scoverage-plugin` :( https://mvnrepository.com/artifact/org.scoverage/scalac-scoverage-plugin

https://github.com/scoverage/scalac-scoverage-plugin/issues/339

**Comment by rtitle** [comment]
Yay https://github.com/scoverage/scalac-scoverage-plugin/pull/341

**Comment by rtitle** [comment]
fiab-start failed, not sure why.. jenkins retest

**Review by rtitle — APPROVED** [review]

**Pull request merged**